bears/go/GoErrCheckBear.py

```diff
@@ -0,0 +1,18 @@
+from coalib.bearlib.abstractions.Linter import linter
+
+
+@linter(executable='errcheck',
+        output_format='regex',
+        output_regex=r'(?P<filename>[^:]+):(?P<line>\d+):'
+        r'(?P<col>\d+)\s*(?P<message>.*)',
+        result_message="Erroneous function calls detected by errcheck")
```

Comment:
I don't htink the message is particularly good, maybe rather something like "This function call has an unchecked error."?